CMakeLists.txt

```diff
@@ -33,31 +33,57 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR})
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR})
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/cmake)
 
+if(WIN32)
+  set(WIN_EXTRA_LIBS_PATH "C:/Programs" CACHE PATH
+    "Base path to extra libs/headers on Windows (SDL, dirent, pthreads, msinttypes folders should be in here).")
+  list(APPEND CMAKE_PREFIX_PATH "${WIN_EXTRA_LIBS_PATH}" "${WIN_EXTRA_LIBS_PATH}/SDL")  # hints for FindSDL
+endif()
+
 find_package(Qt5Widgets)
 find_package(Qt5Xml)
 find_package(Qt5LinguistTools)
 find_package(Qt5PrintSupport)
 find_package(Qt5Multimedia)
 
-message(STATUS "Looking for SDL ")
-find_package(Sdl)
-if(SDL_FOUND)
-  message("SDL Include Path: " ${SDL_INCLUDE_DIR})
+if(Qt5Widgets_FOUND)
+  message(STATUS "Qt Version: ${Qt5Widgets_VERSION}")
 else()
-  message("SDL not found! Joysticks support will be disabled. Simulator audio will not work.")
+  message(WARNING "Qt not found! Companion and Simulator builds disabled.")
 endif()
 
-message(STATUS "Looking for FOX ")
-find_package(Fox)
```

Comment:
That  script seems to conform to the standard CMake module conventions and doesn't output any messages on its own.  I tried a few things with it but for some reason it would always report negative even with QUIET flag.  I tried a few other FindPkg modules from CMake and most honor the QUIET correctly, but some don't.  Yet I can't find a difference between them (or FindFox) which explains it.   :imp: